docs/src/design/decoder/main.md

```diff
@@ -348,14 +354,19 @@ When the VM executes a `DYN` operation, it does the following:
 Before an `END` operation is executed by the VM, the prover populates $h_0, ..., h_3$ registers with the hash of the block which is about to end. The prover also sets values in $h_4$ and $h_5$ registers as follows:
 * $h_4$ is set to $1$ if the block is a body of a *loop* block. We denote this value as `f0`.
 * $h_5$ is set to $1$ if the block is a *loop* block. We denote this value as `f1`.
+* $h_6$ is set to $1$ if the block is a *call* block. We denote this value as `f2`.
+* $h_7$ is set to $1$ if the block is a *syscall* block. We denote this value as `f3`.
 
 ![decoder_end_operation](../../assets/design/decoder/decoder_end_operation.png)
```

Comment:
I think this picture needs to be updated to include `f2` and `f3` flags.

Reply:
The problem though is that we don't include the excalidraw files in the repo, so to make I change to the picture, I have to redo it all. Do you have that one still somewhere?

Also I will include the excalidraw files for the new diagrams I just created

Reply:
Good point! Not sure what's the best solution here - but here is my old excalidraw file for the decoder section (had to zip it because github does not allow uploading this file format).

[decoder.excalidraw.zip](https://github.com/user-attachments/files/17478993/decoder.excalidraw.zip)

Reply:
Removed the excalidraw files, and updated the END operation image with `f2` and `f3`